examples/message/send_receive/SendReceive.cs

```diff
@@ -0,0 +1,153 @@
+﻿using System;
+using NationalInstruments.SystemLink.Clients.Core;
+using NationalInstruments.SystemLink.Clients.Message;
+using Newtonsoft.Json;
+
+namespace NationalInstruments.SystemLink.Clients.Examples.Message
+{
+    /// <summary>
+    /// Example for the SystemLink Message client API that opens two message
+    /// sessions, sends JSON messages from one, then reads those messages back
+    /// from the other.
+    /// </summary>
+    class SendReceive
+    {
+        static void Main(string[] args)
+        {
+            /*
+             * See the configuration example for how a typical application
+             * might obtain a configuration.
+             */
+            var configuration = ExampleConfiguration.Obtain(args);
+
+            /*
+             * Open a session and subscribe to a couple of topics. Once
+             * subscribed, the server will queue messages published to any of
```

Comment:
The wording in the last half of this sentence seems a bit off to me. Maybe something like "Once subscribed, the server will begin to queue messages published to these topics." (Feel free to ignore this suggestion, but I think it still needs to be re-worded).